trpc/router/feedback.ts

```diff
@@ -0,0 +1,28 @@
+import { protectedProcedure, router } from '../trpcBackend';
+import { createFeedbackSchema } from '../../src/schema/feedback';
+
+export const feedback = router({
+    create: protectedProcedure.input(createFeedbackSchema).mutation(async ({ ctx, input }) => {
+        if (!process.env.FEEDBACK_URL) {
+            return;
+        }
+        const { title, description, href } = input;
```

Comment:
You can do inline on 5 line, like `({ ctx, input: { title, description, href }})`